inngest/_internal/env_lib.py

```diff
@@ -18,12 +18,28 @@ def get_environment_name() -> typing.Optional[str]:
     )
 
 
-def is_true(env_var: const.EnvKey) -> bool:
+def get_url(env_var: const.EnvKey) -> typing.Optional[str]:
+    """
+    Get a URL from an env var. Returns None if the env var is not set or if its value is not a valid URL
+    """
+
+    val = os.getenv(env_var.value)
+    if val is None:
+        return None
+
+    parsed = net.parse_url(val)
+    if isinstance(parsed, Exception):
+        return None
+
+    return parsed
+
+
+def is_truthy(env_var: const.EnvKey) -> bool:
     val = os.getenv(env_var.value)
     if val is None:
         return False
 
-    if val.lower() in ("true", "1"):
-        return True
+    if val.lower() in ("false", "0"):
+        return False
 
```

Comment:
TS whitelists truthy values instead atm (inngest/inngest-js#488):
```ts
if (["true", "1"].includes(loweredAndTrimmedValue)) {
  return true;
}
```
Can we do the same here? Want to make sure that e.g. `INNGEST_DEV=somethingrandom` works the same in both.

Reply:
Leaving; TS will also do this in v4 to match the default-to-Cloud method.